cmd/lotus-storage-miner/market.go

```diff
@@ -40,29 +48,115 @@ var disableCmd = &cli.Command{
 	},
 }
 
-var setPriceCmd = &cli.Command{
-	Name:  "set-price",
-	Usage: "Set price that miner will accept storage deals at (FIL / GiB / Epoch)",
-	Flags: []cli.Flag{},
+var setAskCmd = &cli.Command{
+	Name:  "set-ask",
+	Usage: "Configure the miner's ask",
+	Flags: []cli.Flag{
+		&cli.Uint64Flag{
+			Name:     "price",
+			Usage:    "Set the price of the ask (specified as FIL / GiB / Epoch) to `PRICE`",
+			Required: true,
+		},
+		&cli.Uint64Flag{
+			Name:        "duration",
+			Usage:       "Set the duration (specified as epochs) of the ask to `DURATION`",
```

Comment:
Is that the maximum or the minimum deal duration we'll accept?

Reply:
No. This wording _is_ confusing. I've since updated it as per your feedback.

The duration is the quantity of epochs from the current chain head (when the `set-ask` command is run) at which point the ask will expire.